src/main/java/com/ciandt/techgallery/service/impl/EndorsementServiceImpl.java

```diff
@@ -280,7 +280,7 @@ public Endorsement addOrUpdateEndorsementPlusOne(EndorsementResponse endorsement
     entity.setTechnology(Ref.create(technology));
     entity.setActive(true);
     endorsementDao.add(entity);
-    notifyEndorsedUser(tgEndorserUser, tgEndorsedUser, technology);
+    //notifyEndorsedUser(tgEndorserUser, tgEndorsedUser, technology);
     UserProfileServiceImpl.getInstance().handleEndorsement(entity);
```

Comment:
@tsr-thulio please make sure this is the same code above. If it is, lets refactor.